pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env

```diff
@@ -57,8 +57,9 @@ NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS=3
 PERSONAL_DATA_VAULT_API_KEY=personal-data-vault-api-key
 PERSONAL_DATA_VAULT_API_BASE_PATH="http://pagopa-pdv-mock:8080/tokenizer/v1/transaction/"
 
-AUTHORIZATION_REQUESTED_RETRY_EVENT_BASE_INTERVAL_SECONDS=600
-AUTHORIZATION_REQUESTED_RETRY_EVENT_MAX_ATTEMPTS=3
+AUTHORIZATION_OUTCOME_WAITING_EVENT_BASE_INTERVAL_SECONDS=60
+AUTHORIZATION_OUTCOME_WAITING_EVENT_MAX_ATTEMPTS=3
+AUTHORIZATION_OUTCOME_WAITING_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET=10
```

Comment:
to be aligned to the event dispatcher used ones after name refactoring

Reply:
These already are the name used in event dispatcher after refactoring

Reply:
I was suggesting to change `AUTHORIZATION_OUTCOME_WAITING_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET ` adding the time unit in the env variable name to make it explicit. what are those? seconds, milliseconds?